src/main/webapp/app/exercises/programming/shared/code-editor/ace/code-editor-ace.component.ts

```diff
@@ -178,6 +208,9 @@ export class CodeEditorAceComponent implements AfterViewInit, OnChanges, OnDestr
                 }
             });
         }
+        // Remove open inline feedback widgets for new feedback
+        this.linesWithNewFeedback = [];
+        // this.changeDetectorRef.detectChanges(); // Manually trigger change detection => otherwise Angular might be too smart and not update the DOM
```

Comment:
Should you readd this line or can you delete that?

Reply:
Good catch! It's unnecessary because `linesWithNewFeedback` will never be updated in the same render cycle. Therefore Angular will always properly the related DOM elements. I've removed it.